### PR TITLE
Add __init__.py in some dirs

### PR DESCRIPTION
**Pull request by naeioi**
This PR adds `__init__.py` to all source code directories. This is a workaround for #465 .

### CODE REVIEW

**Comment by ryanjulian** [comment]
Does this add an `__init__.py` to all modules which lack one?

I think this will keep coming up, so it's best to just revert to the old way than be half-old and half-new

**Comment by naeioi** [comment]
An `__init__.py` is now added to all source code directories that miss one.

**Comment by naeioi** [comment]
Coverage decreases and blocks merging. Any idea on how I can solve this?

File: garage/tf/__init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+"""Tensorflow Branch."""
```
[the remark this answers is not in the document]
Comment (naeioi) [reply garage/tf/__init__.py]: It's not only algorithm but also includes baseline, sampler, space, etc. Maybe TensorFlow Modules?

[the remark this answers is not in the document]
Comment (ryanjulian) [reply garage/tf/__init__.py]: sure

TensorFlow Algorithm Library

**Comment by ryanjulian** [comment]
I'm also curious why CodeCov is not reporting. Maybe @gonzaiva 's new change messed it up?

**Comment by ryanjulian** [comment]
It looks like coverage may have legitimately dropped in stateful_pool.py https://coveralls.io/builds/21218982/source?filename=garage%2Fsampler%2Fstateful_pool.py

**Comment by naeioi** [comment]
@ryanjulian Coverage is not consistent. It reported `-5.06%` dropping of coverage in `theano/optimizers/conjugate_gradient_optimizer.py` at my first commit.

**Comment by ryanjulian** [comment]
`conjugate_gradient_optimizer.py` is always flaky in coverage because it is called by the algorithm integration tests, but I think there are actual unit test files for `stateful_pool.py`

**Comment by ryanjulian** [comment]
CodeCov didn't receive a report for your build. I think the new Dockerfile may have broken the coverage upload.

https://codecov.io/gh/rlworkgroup/garage/pulls

**Pull request merged**